demo-app/src/components/aruba/ArubaPin.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Box, Text, ThemeContext } from 'grommet';
+import { normalizeTheme } from 'grommet-leaflet-core/src/utils';
```

Comment:
If we are expecting callers to use this util (which seems reasonable given this use case), we might want to think of a better export structure so they don't have to drill in so far

Reply:
Adjusted export structure in latest commit